app/main.py

```
@@ -576,6 +576,25 @@ async def chat(
             pass
     return chat_message
 
+@app.get("/chat/{chat_id}/image/{image_id}")
+async def chat_image(
+    chat_id: str,
+    image_id: str,
```

Ruff (0.12.2)

Unused function argument: `image_id`

(ARG001)